crates/apub/src/protocol/objects/page.rs

```diff
@@ -124,6 +125,40 @@ impl Attachment {
       _ => None,
     }
   }
+
+  pub(crate) fn as_markdown(&self) -> String {
+    match self {
+      Attachment::Image(i) => {
+        format!(
+          "![{}]({})",
+          i.name.as_ref().unwrap_or(&String::default()),
```

Comment:
String::default is an empty string so the link would be unclickable. Better use a string "Attachment" as [translation](https://github.com/LemmyNet/lemmy-translations/blob/main/translations/en.json).

Reply:
This is an image, the text isn't suppose to be displayed.

Reply:
Youre right, in that case the url should be passed through `proxy_image_link()`.